.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,46 @@
+# NOTE: This name appears in GitHub's Checks API and in workflow's status badge.
+name: ci-build
+
+# Trigger the workflow when:
+on:
+  # A push occurs to one of the matched branches.
+  push:
+    branches: [stable, master]
+  # Or when a pull request event occurs for a pull request against one of the
+  # matched branches.
+  pull_request:
+    branches: [stable, master]
```

Comment:
```yaml
# Trigger the workflow when:
on:
  # A push occurs to one of the matched branches.
  push:
    branches:
      - master
      - stable/*
  # Or when a pull request event occurs for a pull request against one of the
  # matched branches.
  pull_request:
    branches:
      - master
      - stable/*
```
We name stable branches with `stable/` prefix, so we should use something like in Oasis Core.